src/Action/EntrypointAction.php

```diff
@@ -9,12 +9,12 @@
  * file that was distributed with this source code.
  */
 
-namespace ApiPlatform\Core\Hydra\Action;
+namespace ApiPlatform\Core\Action;
```

Comment:
Good catch. But HAL has it: http://haltalk.herokuapp.com/explorer/browser.html#/